### PR TITLE
delete-note

### PR DESCRIPTION
**Pull request by NoahBovee**

### CODE REVIEW

File: src/components/CreateArea.jsx [hunk 1]
```diff
@@ -61,9 +75,33 @@ const CreateArea = ({ onAdd }) => {
...
+                className="my-1 mx-1 mb-2 is-size-6"
+                value={note.category}
+                type="text"
+                placeholder="New category"
+              />
+
               {error && (
                 <p className="help is-danger">
-                  Title and content cannot be empty!
+                  Title, content or category cannot be empty!
```
Comment (CCornez) [inline src/components/CreateArea.jsx]: don't forget to change the message as well :D

File: src/components/CreateArea.jsx [hunk 2]
```diff
@@ -21,11 +23,23 @@ const CreateArea = ({ onAdd }) => {
 
   const submitButton = (event) => {
     event.preventDefault();
-    if (note.title.trim() !== "" && note.content.trim() !== "") {
+    if (
+      note.title.trim() !== "" &&
+      note.content.trim() !== "" &&
+      note.category.trim() !== ""
```
Comment (CCornez) [inline src/components/CreateArea.jsx]: category not required :)

**Review by CCornez — CHANGES_REQUESTED** [review]
Added categories as well ^^

**Comment by CCornez** [comment]
[delete functionality 'notes'](https://trello.com/c/o2Kvc5N2/80-delete-functionality-notes)

**Review by gasparsaldanas — APPROVED** [review]

**Review by CCornez — APPROVED** [review]
Perfect 👌

**Pull request merged**